cypress/support/consts.ts

```diff
@@ -26,3 +26,10 @@ export const ODFCatalogSource = {
     publisher: 'Red Hat',
   },
 };
+/* OCS_SC_STATE is divided up into sub-strings for better readability
+* as OCS_SC_JSONPATH uses some escape sequences in order to fetch the
+* storage cluster's status.
+*/
+const OCS_SC_JSONPATH="{range .items[*]}{.metadata.name==\"ocs-storagecluster\"}{.status.phase}{\"\\n\"}{end}";
```

Comment:
```suggestion
const SC_PHASE_QUERY=(name) => "{range .items[*]}{.metadata.name==${name}"}{.status.phase}{\"\\n\"}{end}";
```

I think we could benefit from changing it into a pure function. Pass the name of storagecluster to get the reqd query